src/DropdownButton.tsx

```diff
@@ -2,6 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import Dropdown from './Dropdown';
+import DropdownToggle, { PropsFromToggle } from './DropdownToggle';
+import { BsPrefixRefForwardingComponent } from './helpers';
+import DropdownMenu from './DropdownMenu';
+
+export interface DropdownButtonProps
+  extends React.PropsWithChildren<PropsFromToggle> {
+  id?: string;
+  title: React.ReactNode;
+  menuRole?: string;
+  renderMenuOnMount?: boolean;
+  rootCloseEvent?: 'click' | 'mousedown';
+  bsPrefix?: string;
+}
+
+type DropdownButton = BsPrefixRefForwardingComponent<
```

Comment:
this isn't relevant here since it doesn't accept `as`, the name of this helper is a bit misleading